.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 sudo: false
-cache: pip
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/virtualenv
```

Comment:
the `--download-cache` options is also deprecated from pip>=6.0

On 13 October 2015 at 07:04, Daniel Roberts <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/dimagi/commcare-hq/pull/8590#discussion_r41827267>:
>
> > @@ -1,6 +1,9 @@
> >  language: python
> >  sudo: false
> > -cache: pip
> > +cache:
> > +  directories:
> > +    - $HOME/.cache/pip
> > +    - $HOME/virtualenv
>
> I think I don't understand the difference between caching .cache/pip and
> using a download cache. If we cache both when only one is necessary then
> that's more stuff to copy on startup, so it's unclear to me what the
> tradeoff is. What's your take on it? For now, I think #8608
> <https://github.com/dimagi/commcare-hq/pull/8608> is good start?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8590/files#r41827267>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa